Tests/ArgumentParserExampleTests/MathExampleTests.swift

```diff
@@ -29,7 +29,7 @@ final class MathExampleTests: XCTestCase {
     let helpText = """
         OVERVIEW: A utility for performing maths.
 
-        USAGE: math <subcommand>
+        USAGE: math <add|multiply|stats>
```

Comment:
Likewise, it seems like this loses the context that add/multiply/stats are subcommands.